### ISSUE TITLE
Missing Market data values shows up as NaN

### ISSUE THREAD

**Issue by jiteshnambiar** [omdv/robinhood-portfolio#7]
In the original repo, historical prices are fetched from morning star and this api has been deprecated. After replacing this with 'tiingo' & providing it with a valid 'TIINGO_API_KEY', it is able to fetch the market data between the request start & end dates, but once this data is written to hdf datafile & read it gets turned into NaN.

This impacts most calculations especially Risk Analysis which relies heavily on the market data.

I'm unable to debug if this is due to the data format returned by tiingo or something inherent in the code base. Kindly suggest how to fix this issue.

**Comment by omdv** [comment]
Let me take a look - may take few days. I am having unexpected issues reproducing python dev environment after OS upgrade.

**Comment by jiteshnambiar** [comment]
Thank you for the prompt reply, but I managed to fix the issue.

The first issue was the column names (from tiingo) were not in lower case and I fixed that by renaming the columns
 + df.rename(columns={'Open': 'open', 'High': 'high', 'Low': 'low', 'Close': 'close', 'Volume': 'volume'}, inplace=True)

Other issue was 'Date' being set as index without being dropped, causing the values to shift making Close values being replaced with Volumes. To fix that, I added the drop parameter.
 + df.set_index('Date', inplace=True, drop=True)
